syncano/models/fields.py

```diff
@@ -687,10 +686,33 @@ def to_native(self, value):
         return value
 
 
-GeoPointStruct = namedtuple('GeoPointHelper', ['latitude', 'longitude'])
+class GeoPoint(Field):
 
+    KILOMETERS = '_in_kilometers'
+    MILES = '_in_miles'
 
-class GeoPoint(Field):
+    class GeoPointStruct(object):
+
+        def __init__(self, latitude=None, longitude=None, distance=None, unit=None):
+            if not latitude and not longitude:
```

Comment:
So you are allowed to have latitude without longitude and vice versa ?
You are not allowed to have latitude=0 and longitude=0 ?